contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -979,7 +979,39 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 	    AND CAST(ao.is_ms_shipped as INT) = 0;      
 
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
-	
+
+/*
+* VIEW_TABLE_USAGE
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.view_table_usage
+AS SELECT DISTINCT CAST(sys.db_name() AS sys.nvarchar(128)) AS "VIEW_CATALOG",
+    CAST((select orig_name from sys.babelfish_namespace_ext where nv.nspname = nspname) AS sys.nvarchar(128)) AS "VIEW_SCHEMA",
+    CAST(v.relname AS sys.sysname) AS "VIEW_NAME",
+    CAST(sys.db_name() AS sys.nvarchar(128)) AS "TABLE_CATALOG",
+    CAST((select orig_name from sys.babelfish_namespace_ext where nt.nspname = nspname) AS sys.nvarchar(128)) AS "TABLE_SCHEMA",
+    CAST(t.relname AS sys.sysname) AS "TABLE_NAME"
+   FROM pg_catalog.pg_namespace nv,
```

Comment:
Please use join of pg_namespace_ext and babelfish_namespace_ext instead of pg_namespace + nspname query.